### ISSUE TITLE
Yum issue while installing isulad

### ISSUE THREAD

**Issue by varanasiroshan2001** [openeuler-mirror/openEulerCodeHeat#9]
**Describe the bug**
While i am installing the isulad container engine, the things we write in the yum file isnt working, its telling that it failed to get any thing named isulad from the openeuler repository. The baseURL might be the issue. I tried with the gpg check key to be imported but the key you provided is coming with the error of "import read failed" or due to the not providing of the mirror list. So, due to there the isulad container engine installation work is stopped.

**To Reproduce**
1. Go to 'A VM in AWS or any linux OS then go to root user'
2. Click on 'the /etc/yum.repos.d/*.repo file and paste the baseurl and other things'
3. Scroll down to 'the terminal and write the yum install isulad -y then it will show the error' below
![image](https://github.com/openeuler-mirror/openEulerCodeHeat/assets/64471180/1e764138-aaf8-4f3e-ad8e-efb58c5446a2)

After that as i do with the gpg alternative you provided it shows read failed error
![image](https://github.com/openeuler-mirror/openEulerCodeHeat/assets/64471180/3366520e-2c65-4426-b127-c93f3a50e0c6)

**Expected behavior**
You will not be able to install the isulad 

**Screenshots**
Added up above

**Desktop (please complete the following information):**
 - OS: [e.g. iOS]  Windows
 - Browser [e.g. chrome, safari] Chrome
 - Version [e.g. 22]  Latest

**Comment by jimmieme** [comment]
@varanasiroshan2001 Could you provide the content of your repo file (etc/yum.repos.d/*.repo) which you add openEuler
 baseurl into ? It looks like the baseurl is wrong. And also if you could provide the url of website you refer to, it will be easier to let me help you.

- This is wrong: `https://repo.openeuler.org/openEuler-22.03-LTS/os/\x86_64/repodata/repomd.xml` 
- This is right:  `https://repo.openeuler.org/openEuler-22.03-LTS/OS/x86_64/repodata/repomd.xml`

**Comment by varanasiroshan2001** [comment]
I got this from your official installation links only which you gave in the Codeheat ISSUE of installing isulad and when i went for that page this was the repo code in that read me file from 
https://github.com/openeuler-mirror/iSulad?tab=readme-ov-file#getting-started I guess there is a small backslash issue before $basearch  in that file due to which this bug rose yes

![image](https://github.com/openeuler-mirror/openEulerCodeHeat/assets/64471180/a93f082d-eec5-4fe4-be18-b5ae85b4d09f)

**Comment by jimmieme** [comment]
> I got this from your official installation links only which you gave in the Codeheat ISSUE of installing isulad and when i went for that page this was the repo code in that read me file from https://github.com/openeuler-mirror/iSulad?tab=readme-ov-file#getting-started I guess there is a small backslash issue before $basearch in that file due to which this bug rose yes

@varanasiroshan2001 Thank you for your information.

You could try write openEuler.repo file by running the cat command, and not by copying and pasting the content of [openEuler] into `openEuler.repo`. Otherwise, you will got `\$` in your openEuler.repo file which result the above error `Error: Failed to download metadata for repo 'openEuler':`

After you rewrite openEuler.repo, you could try run `yum install -y iSulad` again.

**Comment by varanasiroshan2001** [comment]
Yes, I got your point but initially i did that via the cat and EOF command that was given that too didnt search it so thats why tried this instead

**Comment by varanasiroshan2001** [comment]
I did exactly what you said via the cat command but check the below images, its still not getting installed but this time the baseurl issue got over but here the rest of the issues its not solving

![image](https://github.com/openeuler-mirror/openEulerCodeHeat/assets/64471180/6abeaa60-cc80-4d59-9469-48f19f9d13d8)
![image](https://github.com/openeuler-mirror/openEulerCodeHeat/assets/64471180/22873d65-7d7f-43bb-9c11-1fb52010d90a)
